docs/src/main/sphinx/connector/hive-azure.rst

```diff
@@ -2,14 +2,14 @@
 Hive connector with Azure Storage
 =================================
 
-The :doc:`hive` can be configured to query
-Azure Standard Blob Storage and Azure Data Lake Storage Gen2 (ABFS). Azure Blobs
-are accessed via the Windows Azure Storage Blob (WASB). This layer is built on
-top of the HDFS APIs and is what allows for the separation of storage from the
-cluster.
-
-Trino supports both ADLS Gen1 and Gen2. With ADLS Gen2 now generally available,
-we recommend using ADLS Gen2. Learn more from `the official documentation
+The :doc:`hive` can be configured to query Azure Standard Blob Storage and Azure
+Data Lake Storage Gen2 (ABFS). Azure Blobs are accessed via the Windows Azure
+Storage Blob (WASB). This layer is built on top of the HDFS APIs and is what
+allows for the separation of storage from the cluster.
```

Comment:
I think this intro paragraph confuses the subject.  Basically, I think we want to say:
* You can configure our system to read datain Azure Blob Storage
* We support ABFS for reading the current Gen2 system
* We support the legaxy WSAB and ADL system, but they will be dropped at some point so they should migrate

Reply:
Also the naming microsoft uses is a mess.  I think this phrase from [this](https://learn.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction) doc more clearly describes the terms:
> Azure Data Lake Storage Gen2 is a set of capabilities dedicated to big data analytics, built on [Azure Blob Storage](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction).

So ADLS gen 2 is system for mapping big data analytics (meaning Hive/Hadoop/Spark) onto Azure Blob Storage.

Reply:
Hi @dain, I made changes based on your feedback. Let me know if you believe it needs further clarification.

@mosabua I changed the commit message as you requested.